psalm.xml

```diff
@@ -125,6 +163,19 @@
 
         <MissingReturnType errorLevel="info" />
 
+        <MissingTemplateParam>
+            <errorLevel type="suppress">
+                <file name="src/Sylius/Bundle/CoreBundle/Form/DataTransformer/ProductTaxonToTaxonTransformer.php" />
+                <file name="src/Sylius/Bundle/CoreBundle/Form/DataTransformer/ProductVariantsToCodesTransformer.php" />
+                <file name="src/Sylius/Bundle/CoreBundle/Form/DataTransformer/ProductsToCodesTransformer.php" />
+                <file name="src/Sylius/Bundle/CoreBundle/Form/DataTransformer/TaxonsToCodesTransformer.php" />
+                <file name="src/Sylius/Bundle/ProductBundle/Form/DataTransformer/ProductVariantToProductOptionsTransformer.php" />
+                <file name="src/Sylius/Bundle/ProductBundle/Form/DataTransformer/ProductsToProductAssociationsTransformer.php" />
+                <file name="src/Sylius/Bundle/PromotionBundle/Form/Type/PromotionCouponToCodeType.php" />
+                <file name="src/Sylius/Bundle/UserBundle/Form/UserVerifiedAtToBooleanTransformer.php" />
+            </errorLevel>
+        </MissingTemplateParam>
+
```

Comment:
Generics in DataTransformers appeared in Symfony 6.2, until we drop `5.4` support we have to leave this suppression.